server/openapi_server/utils/annotator_client.py

```diff
@@ -0,0 +1,26 @@
+import requests
```

Comment:
I'm not a huge fan of the utils class as it's often used as the default place to store code that one doesn't know where to place. We can keep as it is for now and revisit later.